### ISSUE TITLE
[1337x] Exception (1337x): Le nom distant n'a pas pu être résolu: '1337x.to': An error occurred while sending the request. (Test)

### ISSUE THREAD

**Issue by JuxTh** [Jackett/Jackett#6651]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue. Duplicating issues makes it more difficult for everyone to follow. Your issue may have already been solved in the past as well.

Provide a description of the feature request or bug, the more details the better.

If you are experiencing an issue with a tracker, then:
1. Use your browser to check you can access the site directly, and if a login is required, check you can login and that you do not have any outstanding account issues.
2. If you haven't already, try upgrading to the latest version of Jackett, your issue may have already been resolved.
3. If it is still not working for you, then a **full enhanced log must be included**. Instructions for obtaining logs are here: https://github.com/Jackett/Jackett#troubleshooting

(At the bottom of your Jackett Dashboard page) **Jackett version**:  
(if you installed using Jackett.Binaries.Mono.tar.gz) **Mono version**:

-------------------------------

**Comment by garfield69** [comment]
there is nothing wrong with https://1337x.to/ so this looks like an issue with your DNS

most likely your ISP is blocking this site (see if your browser can access the site)
or
you have this site's address blacklisted (for example some ad prevention tool, or virus checker or firewall)
or
your DNS is not accessible by your computer for some reason (try switching to another DNS)
or
I have seen docket images have trouble accessing DNS after an upgrade (try following https://development.robinwinslow.uk/2016/06/23/fix-docker-networking-dns/ to helps to resolve).

basically nothing to do with Jackett itself.
